src/EnergyPlus/HVACMultiSpeedHeatPump.cc

```diff
@@ -3834,73 +3718,42 @@ namespace HVACMultiSpeedHeatPump {
         int InletNode;             // MSHP air inlet node
         Real64 OutsideDryBulbTemp; // Outdoor dry bulb temperature [C]
         Real64 AirMassFlow;        // Air mass flow rate [kg/s]
-        int FanInletNode;          // MSHP air outlet node
-        int FanOutletNode;         // MSHP air inlet node
         Real64 SavePartloadRatio;
         Real64 SaveSpeedRatio;
         Real64 QCoilActual;  // coil load actually delivered returned to calling component
         Real64 MinWaterFlow; // minimum water flow rate
         Real64 ErrorToler;   // supplemental heating coil convergence tollerance
 
-        auto &MSHeatPump = state.dataHVACMultiSpdHP->MSHeatPump;
+        auto &MSHeatPump = state.dataHVACMultiSpdHP->MSHeatPump(MSHeatPumpNum);
 
-        OutletNode = MSHeatPump(MSHeatPumpNum).AirOutletNodeNum;
-        InletNode = MSHeatPump(MSHeatPumpNum).AirInletNodeNum;
-        if (MSHeatPump(MSHeatPumpNum).DXHeatCoilIndex > 0) {
-            if (state.dataDXCoils->DXCoil(MSHeatPump(MSHeatPumpNum).DXHeatCoilIndex).IsSecondaryDXCoilInZone) {
-                OutsideDryBulbTemp = state.dataZoneTempPredictorCorrector
-                                         ->zoneHeatBalance(state.dataDXCoils->DXCoil(MSHeatPump(MSHeatPumpNum).DXHeatCoilIndex).SecZonePtr)
-                                         .ZT;
+        OutletNode = MSHeatPump.AirOutletNodeNum;
+        InletNode = MSHeatPump.AirInletNodeNum;
+        if (MSHeatPump.DXHeatCoilIndex > 0) {
+            if (state.dataDXCoils->DXCoil(MSHeatPump.DXHeatCoilIndex).IsSecondaryDXCoilInZone) {
+                OutsideDryBulbTemp =
+                    state.dataZoneTempPredictorCorrector->zoneHeatBalance(state.dataDXCoils->DXCoil(MSHeatPump.DXHeatCoilIndex).SecZonePtr).ZT;
             } else {
                 OutsideDryBulbTemp = state.dataEnvrn->OutDryBulbTemp;
             }
-        } else if (MSHeatPump(MSHeatPumpNum).DXCoolCoilIndex > 0) {
-            if (state.dataDXCoils->DXCoil(MSHeatPump(MSHeatPumpNum).DXCoolCoilIndex).IsSecondaryDXCoilInZone) {
-                OutsideDryBulbTemp = state.dataZoneTempPredictorCorrector
-                                         ->zoneHeatBalance(state.dataDXCoils->DXCoil(MSHeatPump(MSHeatPumpNum).DXCoolCoilIndex).SecZonePtr)
-                                         .ZT;
+        } else if (MSHeatPump.DXCoolCoilIndex > 0) {
+            if (state.dataDXCoils->DXCoil(MSHeatPump.DXCoolCoilIndex).IsSecondaryDXCoilInZone) {
+                OutsideDryBulbTemp =
+                    state.dataZoneTempPredictorCorrector->zoneHeatBalance(state.dataDXCoils->DXCoil(MSHeatPump.DXCoolCoilIndex).SecZonePtr).ZT;
             } else {
                 OutsideDryBulbTemp = state.dataEnvrn->OutDryBulbTemp;
             }
         } else {
             OutsideDryBulbTemp = state.dataEnvrn->OutDryBulbTemp;
         }
-        FanOutletNode = MSHeatPump(MSHeatPumpNum).FanOutletNode;
-        FanInletNode = MSHeatPump(MSHeatPumpNum).FanInletNode;
```

Comment:
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:3902]:(style),[unreadVariable],Variable 'FanOutletNode' is assigned a value that is never used.
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:3903]:(style),[unreadVariable],Variable 'FanInletNode' is assigned a value that is never used.